wrapper.sh

```diff
@@ -9,4 +9,4 @@ dry_run=$5
 
 test -e .clang-format || ln -s -f $config .clang-format
 
-$binary --fcolor-diagnostics --Werror $dry_run $infile > $outfile
+$binary --fcolor-diagnostics --color=true --Werror $dry_run $infile > $outfile
```

Comment:
```suggestion
$binary --color=true --Werror $dry_run $infile > $outfile
```

Is `--fcolor-diagnostics` still necessary if color is always enabled?

Reply:
I think we can remove it. I just wasn't sure if it served any purpose I don't know about. For me it looks like it was never impacting anything.